### DISCUSSION TITLE
connect remotely

### DISCUSSION THREAD

**Discussion by RuanMD** [nocodb/nocodb#5084] [Q&A]
I created the postgree database in nocodb, how can I get its credentials to connect remotely?

**Comment by o1lab** [comment]
If you meant created a project within nocodb - its accessible via API. And the database credentials are the same as you provided in NC_DB.

**Reply by RuanMD** [reply]
and how do i find out the host? to be able to do remote access

I put the credentials but I think I need to expose
![image](https://user-images.githubusercontent.com/44640884/218505890-b9db4542-cb45-496a-ab62-150a8081ff17.png)
